etl/etl.rb

```diff
@@ -0,0 +1,15 @@
+require 'byebug'
+class ETL
+  def self.transform(old)
+    new_hash = old.invert
+    result = {}
+    new_hash.keys.each do |element|
+      element.each do |item|
+        a = {item.downcase => new_hash[element]}
+        result.merge!a
```

Comment:
Not sure it's correct syntax or not. If it's correct. Please make it readability: `result.merge! a`